src/app/inbox/components/notification/notification.component.html

```diff
@@ -35,6 +35,11 @@
   [hidden]="!showNotificationContent"
 >
   <div class="body-container">
+    <app-notification-header
+      *ngIf="togglzEnableNewNotifications"
```

Comment:
Do we need a new togglz with a different name for the header and footer? 
Since "togglzEnableNewNotifications" is confusing because the new notification are not just header and footer but all this module

Reply:
Yes, you alright. I will remove it. it is not necessary here.